### DISCUSSION TITLE
Home team placement for soccer games

### DISCUSSION THREAD

**Discussion by RobertD502** [vasqued2/ha-teamtracker-card#11] [Ideas]
First of all, thanks for all the great work!

I was wondering what your thoughts were about reversing the side which teams are shown on the card for soccer games. As it stands, the away team is placed on the left-hand side which is opposite of how it should be - I realize leagues such as the NFL place the away team on the left and the home team on the right. As you can see below, Real Madrid is the away team and should be placed on the right-hand side:

![image](https://user-images.githubusercontent.com/52541649/197879060-54ec11b1-579f-4a82-8ac9-b8e82a1ca09f.png)

This is the ESPN page showing the correct placement:
![image](https://user-images.githubusercontent.com/52541649/197879130-8364debd-0515-4762-94cb-25ca700b037a.png)

Perhaps placement can be set to follow this standard based on the sport type returned by ESPN's API?

**Comment by vasqued2** [comment]
v0.5.2 addresses this.  

If you would prefer that the Home team always be placed on the left or right side (different countries and sports have different standards so there is no universal order that works for everyone), you can now use the `home_side` option in the card configuration to do so.  Valid values are `left` and `right`.

If you don't use the `home_side` option, it will continue to function as it does today and put the team you are tracking first.

Try it and let me know if it works or you have any problems with it.

Example
```
  - type: custom:teamtracker-card
    entity: sensor.tt_test
    outline: true
    outline_color: lightgray
    show_league: true
    home_side: right
```

**Reply by RobertD502** [reply]
@vasqued2 yes, I created this thread before opening the issue which was already addressed.
